### PR TITLE
Grog mpu6050 2

### PR DESCRIPTION
**Pull request by supertick**
working mpu 6050

### CODE REVIEW

File: src/main/java/org/myrobotlab/service/config/Mpu6050Config.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package org.myrobotlab.service.config;
+
+public class Mpu6050Config extends ServiceConfig {
```
Comment (kwatters) [inline src/main/java/org/myrobotlab/service/config/Mpu6050Config.java:3]: I think this config needs the i2c address as well.. but we can do that in a different PR..

Comment (supertick) [reply src/main/java/org/myrobotlab/service/config/Mpu6050Config.java:3]: fixed
